DNN Platform/Library/Services/Mail/SMTPInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+using System.Text;
+
+namespace DotNetNuke.Services.Mail
+{
+    public class SMTPInfo
```

Comment:
I think we should use PascalCase for the class name here.

```suggestion
    public class SmtpInfo
```

Reply:
Resolved.